.github/workflows/branch_deleted.yml

```diff
@@ -0,0 +1,21 @@
+name: Branch deleted
```

Comment:
Ok so here we delete the pre-release whenever a branch gets deleted.

Reply:
Yup, auto cleanup